cms/envs/common.py

```diff
@@ -358,6 +358,9 @@
 # Clickjacking protection can be enabled by setting this to 'DENY'
 X_FRAME_OPTIONS = 'ALLOW'
 
+# Platform for Privacy Preferences header
+P3P_HEADER = 'CP="Open EdX does not have a P3P policy."'
```

Comment:
@douglashall I did a quick search/read on my commute.  If what you say is true, I'd say we shouldn't waste much time on this and I would give it a :+1: .  What do you think?